floodviz/templates/hydrograph.html

```diff
@@ -3,32 +3,84 @@
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}"/>
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}"/>
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/map.css') }}" />
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
 	<title>Flood Viz</title>
 
+	<script type="text/javascript">
+		(function (i, s, o, g, r, a, m) {
+			i['GoogleAnalyticsObject'] = r;
+			i[r] = i[r] || function () {
+					(i[r].q = i[r].q || []).push(arguments)
+				}, i[r].l = 1 * new Date();
+			a = s.createElement(o),
+				m = s.getElementsByTagName(o)[0];
+			a.async = 1;
+			a.src = g;
+			m.parentNode.insertBefore(a, m)
+		})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+		ga('create', 'UA-103831764-1', 'auto');
+		ga('send', 'pageview');
+
+		// Global Namespace for project - FV
+		var FV = FV || {};
+
+		// Wrapper function for Google Analytics explicit send
+		FV.ga_send_event = function(category, action, label) {
+			ga('send', {
+				hitType: 'event',
+				eventCategory: category,
+				eventAction: action,
+				eventLabel: label
+			});
+		}
+
+	</script>
+
 </head>
 
 <body>
 
-	<div class="main-content container-fluid">
-		<div id="hydrograph"><svg></svg></div>
-		<div id="map"></div>
+	<div class="main-content container">
+		<div class="row">
+			<div class="col-sm-12 col-centered">
+				<div id="hydrograph"></div>
+			</div>
+		</div>
 	</div>
 
+	<div id="map"><style>#map {height: 0; width: 0;} #maptip {opacity: 0;}</style></div>
+
+
+
 	<script type="text/javascript">
 		// Global Namespace for project - FV
 		var FV = FV || {};
+		FV.config ={
+			debug: {{ config.DEBUG | tojson | safe }}
+		};
+
 		FV.hydrograph_data_path = "{{ url_for('timeseries_data', filename='data/hydrograph_data.json') }}";
 		FV.hydrograph_dimensions = {{ config.HYDROGRAPH_ASPECT_RATIO | safe }};
 		FV.hydrograph_display_ids = {{ display_sites | safe }};
+
+		FV.mapinfo = {{ mapinfo | tojson | safe }};
 	</script>
 
 	<script type="text/javascript" src="{{ url_for('static', filename='bower_components/jquery/dist/jquery.min.js') }}"></script>
 	<script type="text/javascript" src="{{ url_for('static', filename='bower_components/bootstrap/dist/js/bootstrap.js') }}"></script>
 	<script type="text/javascript" src="{{ url_for('static', filename='bower_components/d3/d3.js') }}"></script>
 	<script type="text/javascript" src="{{ url_for('static', filename='js/hydrograph.js')}}"></script>
+	<script type="text/javascript"
+			src="{{ url_for('static', filename='bower_components/proj4/dist/proj4.js') }}"></script>
+	<script type="text/javascript" src="{{ url_for('static', filename='js/map.js') }}"></script>
+	<script type="text/javascript" src="{{ url_for('static', filename='js/floodviz_onready.js') }}"></script>
```

Comment:
I don't think you should be bringing in floodviz_onready.js. You only need to create the hyrograph. I think each of this "embeddable" views will have their own "xxx_onready.js".